docs/build/isc/v1.0.0-rc.6/docs/how-tos/create-foundry.md

```diff
@@ -0,0 +1,69 @@
+---
+description: How to create a L1 foundry
+image: /img/logo/WASP_logo_dark.png
+tags:
+  - foundry
+  - EVM
+  - how-to
+---
+
+## About Foundry
+
+The stardust update allows you to create your own native tokens. Native tokens are minted by a so-called Foundry. The Foundry allows you to specify a max supply once and change the circulating supply. This how-to will explain how to create a L1 foundry from L2.
```

Comment:
```suggestion
The Stardust update allows you to create your own native tokens. Native tokens are minted by a [Foundry](/tips/tips/TIP-0018/#foundry-output). The Foundry allows you to specify a maximum supply once and change the circulating supply. This how-to will explain how to create an L1 foundry from L2.
```

Reply:
Since it's also used in the other foundry related examples, we could probably turn the introduction into a variable as we do with admonitions and then add the last sentence that tells you what the guide will do as a separate paragraph, wdyt?